src/gui/reports/ReportsWidgetHealthcheck.cpp

```diff
@@ -256,6 +290,7 @@ void ReportsWidgetHealthcheck::calculateHealth()
     } else {
         m_referencesModel->setHorizontalHeaderLabels(QStringList() << tr("") << tr("Title") << tr("Path") << tr("Score")
                                                                    << tr("Reason"));
+        m_ui->healthcheckTableView->sortByColumn(0, Qt::AscendingOrder);
```

Comment:
`setSortingEnabled(true);` of line 155 automatically calls `sortByColumn()` and on line 154 we already do `setSortIndicator(0, Qt::AscendingOrder)`
```suggestion
```